ui/components/app/confirm/info/row/copy-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useCallback, useState } from 'react';
+
+import { IconColor } from '../../../../../helpers/constants/design-system';
+import { Icon, IconName, IconSize } from '../../../../component-library';
+
+export const CopyIcon: React.FC<{ copyText: string }> = ({ copyText }) => {
+  const [success, setSuccess] = useState(false);
+
+  const copyToClipboard = useCallback(async () => {
+    try {
+      await navigator.clipboard.writeText(copyText);
+      setSuccess(true);
+    } catch (error: unknown) {
+      console.error(error);
```

Comment:
Should we add a message to this for clarity such as:

`console.error('Failed to copy text to clipboard', error)`

Reply:
I removed try/catch as I refactored it to use hook `useCopyToClipboard`